### PR TITLE
Add documentation for python clickhouse-driver integration

### PR DESCRIPTION
**Pull request by mimre25**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

This adds the documentation for a new python integration for `clickhouse-driver`.

See https://github.com/getsentry/sentry-python/issues/2154.
Companion PR: https://github.com/getsentry/sentry-python/pull/2167

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by shanamatthews** [comment]
Thanks @mimre25 ! I'll let Anton take a first pass at these docs and then I'll review.

**Comment by sentrivana** [comment]
We still haven't gotten around to review the SDK PR. I've added it and this PR to our internal board so that we pick it up soon.

**Review by antonpirker — APPROVED** [review]
lgtm

File: src/platforms/python/common/configuration/integrations/index.mdx [hunk 1]
```diff
@@ -18,6 +18,7 @@ If you don't want integrations to be enabled automatically, set the `default_int
 
 - [ASGI](asgi/)
 - [asyncio](asyncio/)
+- [Clikchouse Driver](clickhouse-driver/)
```
Comment (shanamatthews) [inline src/platforms/python/common/configuration/integrations/index.mdx:21]:
```suggestion
- [clickhouse-driver](clickhouse-driver/)
```

**Review by shanamatthews — APPROVED** [review]
Great job with these docs! Made some minor suggestions that I'll go ahead and incorporate. After that, you're good from the docs side!

File: src/platforms/python/common/configuration/integrations/clickhouse-driver.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Clickhouse Driver
+description: "Learn about importing the Clickhouse Driver integration and how it captures queries from clickhouse-driver as breadcrumbs."
+sidebar_order: 27
```
[the remark this answers is not in the document]
Comment (shanamatthews) [reply src/platforms/python/common/configuration/integrations/clickhouse-driver.mdx:4]: Working fine after a rebuild...

**Pull request merged**